api/src/services/heartbeat-filter.js

```diff
@@ -0,0 +1,11 @@
+module.exports = string => {
+  if(string === 'true') {
+    return 'true';
```

Comment:
Afraid that if you `setInterval` with `'true'` (or `true`, or `false`, or any other string), your function will end up executing at the minimum allowed interval (10ms). I think that we should do the opposite: impose a minimum interval when a truthy NaN is received.